### DISCUSSION TITLE
v0.10.10

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#990] [Releases]
<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.10.10'>v0.10.10</a>.</em>

**Comment by orazioedoardo** [comment]
The changelog should mention that libsignal-client 0.18.1 is required, shouldn't it?

**Reply by AsamK** [reply]
Good point, I've added it, thanks.
